src/TetherComponent.jsx

```diff
@@ -70,42 +67,35 @@ class TetherComponent extends Component {
     renderElementTo: null,
   };
 
+  // The DOM node of the target, obtained using innerRef in the render prop
   _targetNode = null;
```

Comment:
If you use `createRef()` here, you don't need `_getTargetRef`. You just pass `_targetNode` to `ref={...}` and you access the `.current` property on it.

https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

Reply:
I will make that change. [Originally](https://github.com/danreeves/react-tether/pull/194/commits/17cf9dd8735819f8d1368be1eb9d4815bc037e39#diff-22ebbc014eae1795c9d4844dab527e0dR145) `_update` was called after getting the ref, which was why they are functions, but that's not done anymore so I can change it